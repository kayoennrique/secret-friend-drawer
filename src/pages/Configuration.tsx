import Footer from "../components/Footer";
import Form from "../components/Form";
import ListParticipants from "../components/ListParticipants";

const Configuration = () => {
    return (
        <>
            <Form />
            <ListParticipants />
            <Footer />
        </>
    );
}

export default Configuration;