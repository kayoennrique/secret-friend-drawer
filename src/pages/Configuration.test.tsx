import { render } from "@testing-library/react";
import React from "react";
import { RecoilRoot } from "recoil";
import Configuration from "./Configuration";

const mockNavigation = jest.fn();

jest.mock('react-router-dom', () => {
    return{
        useNavigate: () => mockNavigation
    }
});

describe('the configuration page', () => {
    test('must be rendered correctly', () => {
        const { container } = render(
            <RecoilRoot>
                <Configuration />
            </RecoilRoot>
        );
        expect(container).toMatchSnapshot()
    });
});