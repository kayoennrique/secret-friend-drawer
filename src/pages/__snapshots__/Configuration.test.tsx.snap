// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the configuration page must be rendered correctly 1`] = `
<div>
  <form>
    <input
      placeholder="Insira os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
    
  </form>
  <ul />
  <footer>
    <button
      disabled=""
    >
      Iniciar Brincadeira!
    </button>
  </footer>
</div>
`;
